### ISSUE TITLE
Pytorch QAT Accuracy drop after torch.quantization.convert_jit()

### ISSUE THREAD

**Issue by DheerajMadda** [pytorch/pytorch#95287]
Following is the script that I have used to do Quantization Aware Training

#### Step 1. Load fp32 jit model (convolutional model) and prepare the model
trace_jit_model = torch.jit.load("model_jit.pt")
trace_jit_model.to("cuda")
prepare_qat_model = torch.quantization.prepare_jit(trace_jit_model, {"" : torch.quantization.default_qconfig})
#### to retain the grad, so that the params have their grad populated during backward()
for params in prepare_qat_model.parameters():
(indent) params.retain_grad()
prepare_qat_model.train().to("cuda")

#### Step 2. Train the prepared model (Training loop)
Validation Accuracy achieved :- 0.8903

#### Step 3. Convert and save the model
prepare_qat_model.eval().to("cpu")
quantized_qat_jit_model = torch.quantization.convert_jit(prepare_qat_model)

quantized_qat_save_path = os.path.join("model_qat.pt")
torch.jit.save(quantized_qat_jit_model, quantized_qat_save_path)

#### Step 4. Load quantized model and evaluate
Validation Accuracy :- 0.8511

Why there is such a huge drop in accuracy 0.8903 -> 0.8511 before and after convert_jit ?

Pytorch version used:- '1.13.1+cu116'

cc @jerryzh168 @jianyuh @raghuramank100 @jamesr66a @vkuzo @jgong5 @Xia-Weiwen @leslie-fang-intel

**Comment by supriyar** [comment]
it could be due to certain layers being sensitive to quantization. You can try quantizing the model with eager mode and using the Numeric Suite (https://pytorch.org/tutorials/prototype/numeric_suite_tutorial.html) to analyze which layers are sensitive to quantization and either try a different quantization setting for them or running them in higher precision.

Closing the issue, please feel free to post questions to the PyTorch Discussion Forums

**Comment by duanshengliu** [comment]
@DheerajMadda You should try `default_qat_qconfig` instead of `default_qconfig` when using QAT.
